docs/serialization.rst

```diff
@@ -33,46 +34,30 @@ as training. Techncially there are some difficulties with this approach:
 * We do not want to serialize the training data kept in memory, since this can
   be prohibitively large.
 
-Blocks addresses these problems by using a pickling extension called Dill,
-avoiding certain data structures such as generators and nested classes (see the
-:ref:`developer guidelines <serialization_guidelines>`), and by overriding the
-pickling behavior of datasets.
+Blocks addresses these problems by avoiding certain data structures such as
+generators and nested classes (see the :ref:`developer guidelines
+<serialization_guidelines>`) and overriding the pickling behaviour of some
+objects, making the pickling of the main loop possible.
 
-However, in general you should not rely on this serialization approach for the
-long term saving of your models. Problems that remain are
+However, pickling can be problematic for long-term storage of models, because
 
 * Unpickling depends on the libraries used being unchanged. This means that if
   you updated Blocks, Theano, etc. to a new version where the interface has
-  changed, loading your training progress will fail.
+  changed, loading your training progress could fail.
 * The unpickling of Theano objects can be problematic, especially when
   transferring from GPU to CPU or vice versa.
 * It is not possible on Python 2 to unpickle objects that were pickled in Python
   3.
```

Comment:
Yes, but it doesn't always work out of the box. You might need to specify the encoding as a keyword argument, and you might have problems with bytestrings to unicode conversion and such.